uk_bin_collection/uk_bin_collection/councils/GatesheadCouncil.py

```
@@ -113,69 +113,69 @@ def parse_data(self, page: str, **kwargs) -> dict:
+                    # If no links found, try getting text directly
+                    if not bin_types:
+                        bin_text = bin_cell.get_text(strip=True)
+                        if bin_text:
+                            bin_types = [bin_text]
+                    
+                    # Parse the date
+                    if current_month and day:
+                        try:
+                            # Construct date string: "day month year"
+                            date_str = f"{day} {current_month} {current_year}"
+                            parsed_date = datetime.strptime(date_str, "%d %B %Y")
```

Ruff (0.14.14)

[error] `datetime` may be undefined, or defined from star imports

(F405)